### ISSUE TITLE
[P4] Starting to write C1 and C3 tables. stuck at p4.

### ISSUE THREAD

**Issue by jesse-p** [madMAx43v3r/chia-plotter#472]
environmental information：
      CentOS Linux release 7.9
      gcc version 4.8.5 20150623 (Red Hat 4.8.5-44) (GCC)
      

chia-plot disk failed, there was no response at P4, and no error was reported. It lasted for a long time until the program exited.

This is shown below:
`
[P3-1] Table 4 took 122.078 sec, wrote 3465880239 right entries
[P3-2] Table 4 took 184.406 sec, wrote 3465880239 left entries, 3465880239 final
[P3-1] Table 5 took 116.259 sec, wrote 3532496229 right entries
[P3-2] Table 5 took 194.244 sec, wrote 3532496229 left entries, 3532496229 final
[P3-1] Table 6 took 125.675 sec, wrote 3712776991 right entries
[P3-2] Table 6 took 207.44 sec, wrote 3712776991 left entries, 3712776991 final
[P3-1] Table 7 took 110.294 sec, wrote 4293267089 right entries
[P3-2] Table 7 took 235.142 sec, wrote 4293267089 left entries, 4293267089 final
Phase 3 took 1860.64 sec, wrote 21873540808 entries to final plot
[P4] Starting to write C1 and C3 tables

`

How to solve this problem？

**Comment by therealflinchy** [comment]
Did it eventually finish?

Phase 4 can be incredibly slow, looks like it probably will be for you with that 1800 second p3 time

**Comment by jesse-p** [comment]
> 它最终完成了吗？
> 
> 第 4 阶段可能非常缓慢，看起来可能需要1800 秒 p3 时间

**Comment by jesse-p** [comment]
> Did it eventually finish?
> 
> Phase 4 can be incredibly slow, looks like it probably will be for you with that 1800 second p3 time


no,  I've been waiting for more than ten hours and I'm still in this state.

In CentOS 7 system, it seems that the P4 phase is stuck here all the time, it will not be further executed, and there is no error report
